__tests__/app.test.js

```diff
@@ -14,3 +24,35 @@ describe.only("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Check if returned array has three objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+      });
+  });
+  test("200: Returns an array of topic objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.map((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  test("404: Responds with an error message if topics query is invalid", () => {
```

Comment:
This is just a non-existent route - it's not actually related to the topics endpoint - it's the same as requesting /api/bananas - in which case it can go in its own describe block :) otherwise great test!